src/main/java/com/patchworkmc/annotation/AnnotationProcessor.java

```diff
@@ -49,29 +64,39 @@ public AnnotationVisitor visitAnnotation(String descriptor, boolean visible) {
 			// Ignore Kotlin metadata
 
 			return super.visitAnnotation(descriptor, visible);
-		} else {
-			Patchwork.LOGGER.warn("Unknown class annotation: " + descriptor + " " + visible);
-			return new AnnotationPrinter(super.visitAnnotation(descriptor, visible));
 		}
+
+		return new AnnotationPrinter(super.visitAnnotation(descriptor, visible));
```

Comment:
I liked what you had going with printing Forge annotations, maybe use something like this?
```suggestion
		if (isForgeAnnotation(descriptor)) {
			Patchwork.LOGGER.warn("Unknown Forge class annotation: " + descriptor);
			
			return new AnnotationPrinter(super.visitAnnotation(descriptor, visible));
		}
		
		return super.visitAnnotation(descriptor, visible);
```